test/unit/config.js

```diff
@@ -1,7 +1,7 @@
 var Lab = require('lab');
 var Code = require('code');
 var Proxyquire = require('proxyquire');
-var Config = Proxyquire('../config', {
+var Config = Proxyquire('../../config', {
```

Comment:
here begins all of the path changes caused by restructuring the tests to be under `test/unit`. this allows lab to not grep e2e tests when only caring about unit tests